cilium/cmd/bpf_policy_delete.go

```diff
@@ -27,10 +27,11 @@ var bpfPolicyDeleteCmd = &cobra.Command{
 	PreRun: requireEndpointID,
 	Run: func(cmd *cobra.Command, args []string) {
 		common.RequireRootPrivilege("cilium bpf policy delete")
-		updatePolicyKey(parsePolicyUpdateArgs(cmd, args), false)
+		updatePolicyKey(parsePolicyUpdateArgs(cmd, args, isDeny), false)
 	},
 }
 
 func init() {
+	bpfPolicyDeleteCmd.Flags().BoolVar(&isDeny, "deny", false, "Sets deny mode")
```

Comment:
>This one is a delete command, is it checking for this bit and only deleting the entry if the bit is set? Or does it work another way?

yes, it's exactly like that, it will check if the deny bit is set.

Reply:
OK, then `Sets deny mode` doesn't seem like a very helpful description of the flag. Maybe
```suggestion
	bpfPolicyDeleteCmd.Flags().BoolVar(&isDeny, "deny", false, "Only delete deny policy entries")
```